.chloggen/use-pdata-with-carbonreceiver.yaml

```diff
@@ -0,0 +1,20 @@
+# Use this changelog template to create an entry for release notes.
+# If your change doesn't affect end users, such as a test fix or a tooling change,
+# you should instead start your pull request title with [chore] or use the "Skip Changelog" label.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: enhancement
```

Comment:
Isn't this a breaking change for anyone who was using the changed interfaces?

Reply:
It’s not since the inputs, outputs and configs are unchanged. We keep all of it the same.